docs/sources/v0.47.x/javascript-api/xk6-disruptor/faults/pod-termination.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Pod Termination'
+description: 'xk6-disruptor: Pod Termination Fault attributes'
+weight: 03
+---
+
+# Pod Termination
+
+A Pod Termination Fault allows terminating either a fixed number or a percentage of the pods that matching a selector or back a service.
+
+A Pod Termination fault is defined by the following attributes:
+
+| Attribute     | Type   | Description |
+| ------------- | ------ | --------|
+| count         | integer or percentage | the number of pods to be terminated. It can be specified as a integer number or as a percentage (e.g. `30%`) that defines the fraction of target pods to be terminated|
```

Comment:
> pod-termination.md    15 136 error    Google.Latin Use 'for example' instead of 'e.g.'. (vale-error)

```suggestion
| count         | integer or percentage | the number of pods to be terminated. It can be specified as a integer number or as a percentage, for example `30%`, that defines the fraction of target pods to be terminated|
```